seqr/views/apis/individual_api_tests.py

```diff
@@ -58,7 +72,49 @@ def test_edit_individuals(self):
                             set(response_json['familiesByGuid']['F000001_1']['individualGuids']))
         self.assertListEqual(response_json['familiesByGuid']['F000003_3']['individualGuids'], [])
 
-        self.assertSetEqual({ID_UPDATE_GUID, FAMILY_UPDATE_GUID, CHILD_UPDATE_GUID}, set(response_json['individualsByGuid']))
+        self.assertSetEqual({ID_UPDATE_GUID, FAMILY_UPDATE_GUID, CHILD_UPDATE_GUID},
+                            set(response_json['individualsByGuid']))
         self.assertEqual(response_json['individualsByGuid'][ID_UPDATE_GUID]['individualId'], UPDATED_ID)
         self.assertEqual(response_json['individualsByGuid'][ID_UPDATE_GUID]['maternalId'], UPDATED_MATERNAL_ID)
         self.assertEqual(response_json['individualsByGuid'][CHILD_UPDATE_GUID]['paternalId'], UPDATED_ID)
+
+    def test_delete_individuals(self):
+        individuals_url = reverse(delete_individuals_handler, args=[PROJECT_GUID])
+        _check_login(self, individuals_url)
+
+        # send invalid requests
+        response = self.client.post(individuals_url, content_type='application/json', data=json.dumps({
+            'individualsX': [INDIVIDUAL_IDS_UPDATE_DATA]
+        }))
+        self.assertEqual(response.status_code, 400)
+
+        # send valid requests
+        response = self.client.post(individuals_url, content_type='application/json', data=json.dumps({
+            'individuals': [INDIVIDUAL_IDS_UPDATE_DATA]
+        }))
+        self.assertEqual(response.status_code, 200)
+        response_json = response.json()
+        self.assertListEqual(response_json.keys(), ['individualsByGuid', 'familiesByGuid'])
+
+    def test_individuals_table_handler(self):
+        individuals_url = reverse(receive_individuals_table_handler, args=[PROJECT_GUID])
+        _check_login(self, individuals_url)
+
+        data = 'Family ID	Individual ID	Paternal ID	Maternal ID	Sex	Affected Status	Notes\n\
+"1"	"NA19675"	"NA19678"	"NA19679"	"Female"	"Affected"	"A affected individual, test1-zsf"\n\
+"1"	"NA19678"	""	""	"Male"	"Unaffected"	"a individual note"\n\
+"2"	"HG00733"	""	""	"Female"	"Unaffected"	""'
+
+        f = SimpleUploadedFile("1000_genomes demo_individuals.tsv", data)
+
+        response = self.client.post(individuals_url, {'f': f})
+        self.assertEqual(response.status_code, 200)
+        response_json = response.json()
+        self.assertListEqual(response_json.keys(), ['info', 'errors', 'warnings', 'uploadedFileId'])
+
+        url = reverse(save_individuals_table_handler, args=[PROJECT_GUID, response_json['uploadedFileId']])
+
+        response = self.client.post(url)
+        self.assertEqual(response.status_code, 200)
+        response_json = response.json()
+        self.assertListEqual(response_json.keys(), ['individualsByGuid', 'familiesByGuid'])
```

Comment:
please test some of the data within the individuals/ families to make sure it has the correctly updated data